<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TYSS-Monika" failures="1" tests="3" name="Test" time="8.542" errors="0" timestamp="2024-09-23T13:06:11 IST">
  <testcase classname="Testng_Listner.TestL1" name="testlogo" time="0.051"/>
  <testcase classname="Testng_Listner.TestL1" name="testAppurl" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [tps://opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [tps://opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at Testng_Listner.TestL1.testAppurl(TestL1.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testAppurl -->
  <testcase classname="Testng_Listner.TestL1" name="testtitle" time="0.0">
    <skipped/>
  </testcase> <!-- testtitle -->
</testsuite> <!-- Test -->
